hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/SpawnBalancer.java

```diff
@@ -1225,6 +1306,7 @@ private HostScore calculateHostScore(HostState host, double clusterMaxMeanActive
         double score = 0;
         double meanActive = host.getMeanActiveTasks();
         double usedDiskPercent = getUsedDiskPercent(host);
+        long diskFreeBytes = host.getMax().getDisk() - host.getUsed().getDisk();
```

Comment:
There's a static method already in this class, will use that for now and move later